tests/ClosedXML.Report.Tests/XlsxTemplateTestsBase.cs

```diff
@@ -118,32 +124,36 @@ protected bool WorksheetsAreEqual(IXLWorksheet expected, IXLWorksheet actual, ou
             if (expected.PageSetup.PageOrder != actual.PageSetup.PageOrder)
                 messages.Add("PageOrder differ");
 
-            foreach (var expectedCell in expected.CellsUsed())
+            var usedCells = expected.CellsUsed(true).Select(c => c.Address)
+                .Concat(actual.CellsUsed(true).Select(c => c.Address))
+                .Distinct();
```

Comment:
I am not completely sure, but I guess equal addresses from the different workbooks won't be considered by `Distinct` as the same addresses since they have different `Worksheet` part. Therefore same cells will be processed twice during the comparison. Am I wrong?

Reply:
expected.CellsUsed(true).Select(c => c.Address).Concat(actual.CellsUsed(true).Select(c => c.Address)).Count()
72
expected.CellsUsed(true).Select(c => c.Address).Concat(actual.CellsUsed(true).Select(c => c.Address)).Distinct().Count()
36

Reply:
Right. I've looked into ClosedXML code, and `XLAddress.Equals` method does not compare `Worksheet`. It's fine then.